### DISCUSSION TITLE
Why create account manually for landing zone and the Reference Architecture?

### DISCUSSION THREAD

**Discussion by gruntwork-support** [gruntwork-io/knowledge-base#20] [Help] [answered]
_This message was extracted from a discussion that originally took place in Gruntwork Community Slack. Names and URLs have been removed where appropriate_

**From a customer**

Hello!
Why account creating be part of landing zone or reference architecture? Why do I need to provide account numbers and pre-provision them ?

**Comment by gruntwork-support** [comment, marked as answer]
**From a customer**

There are things delivered as part of the RefArch that need to be bootstrapped and it's easiest to have the accounts already created with Route53 zones already setup.

In particular, you'll need to have the domains, Route53 zones, and accounts pre-created

Gruntwork provides tools to make this bootstrapping process more straight-forward

**Reply by gruntwork-support** [reply]
**From a grunt**

+1 on what person said. The current process for the Reference Architecture is that you create several accounts for us, configure Route 53 in them, and then we deploy everything else into them. We provide the [gruntwork CLI tool](https://github.com/gruntwork-io/gruntwork) to automate much of this process.

In the future, we will switch to having customers use the [account-baseline-root landing zone module](https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/master/modules/landingzone/account-baseline-root) to create the child accounts for the Reference Architecture, but we need to do some more work before that's possible (especially for larger companies, where there are separate owners for account creation, network creation, etc). So for now, the `gruntwork` CLI remains the recommended option for the Ref Arch.

**Reply by gruntwork-support** [reply]
**From a customer**

Can we do on an existing organisation where main/root account is already available ?

**Reply by gruntwork-support** [reply]
**From a grunt**

Yes! You should be creating new accounts that are children of your existing root account.

(using AWS Organizations)

**Reply by drafie** [reply]
Is this still the best practice (using gruntwork CLI tool) at the close of 2022?

**Reply by Etiene** [reply]
Yes, this is still the recommended practice.
